src/main/java/com/aws/greengrass/deployment/DeploymentService.java

```diff
@@ -272,9 +275,59 @@ private Long getPollingFrequency(Topic pollingFrequencyTopic) {
 
     @Override
     protected void shutdown() {
+        saveDeploymentQueueToConfig(); // Save any deployments in queue for next startup
         receivedShutdown.set(true);
     }
 
+    @SuppressWarnings("PMD.AvoidCatchingGenericException")
+    private void saveDeploymentQueueToConfig() {
```

Comment:
nit: use `persist` to be consistent with `persistGroupToRootComponents`, `persistAndPublishDeploymentStatus`, `persistLastSuccessfulDeployment`, etc...

Reply:
agree